python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -112,6 +118,8 @@ def __init__(
             prefix_cluster_info: Whether to add the cluster name to info strs.
             event_summarizer: Utility to consolidate duplicated messages.
             prom_metrics: Prometheus metrics for autoscaler-related operations.
+            gcs_node_info_stub: Stub for interactions with Ray nodes via GCS.
```

Comment:
is it 
> Stub for interactions with Ray nodes via GCS.
or 
> Stub for interactions with Ray nodes via GRPC.
?

Reply:
GCS -- interact with Ray nodes by communicating with the GCS

Reply:
changed to "via gRPC request to the GCS"